ansible/roles/vm_set/tasks/get_dut_port.yml

```diff
@@ -23,3 +22,7 @@
     vmname: "{{ dut_name }}"
   when: external_port is not defined and hostvars[dut_name].type is defined and hostvars[dut_name]['type'] == 'simx'
   become: yes
+
+- set_fact:
+    duts_fp_ports: "{{ duts_fp_ports|default({}) | combine( { dut_name: dut_fp_ports } ) }}"
```

Comment:
@lolyu , here i changed the multi-dut logic a little bit. the duts_fp_ports is a map now.

{dutname: [front panel ports], dutname2: [front panel ports]}

vlan_port module will act as like a single dut module, and we just need to combine them here now. I made this change because in the future, we could have real-dut plus virtual dut as a multi-dut testbed.